src/Microsoft.DocAsCode.Dfm.VscPreview/DfmProcess.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DfmProcess
+{
+    using System;
+    using System.IO;
+    using System.Text;
+
+    using Microsoft.DocAsCode.Dfm;
+    using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.Plugins;
+    class Program
```

Comment:
It is a project which will be compiled to  a .exe file.

---
In reply to: [73886301](https://github.com/dotnet/docfx/pull/583#discussion_r73886301) [](ancestors = 73886301)